spec/unit/procto/to_proc_spec.rb

```diff
@@ -0,0 +1,31 @@
+# encoding: utf-8
+
+require 'spec_helper'
+
+describe Procto, '.to_proc' do
+  include_context 'procto'
+
+  subject { proc.call(text) }
+
+  let(:proc) { klass.to_proc }
+
+  let(:klass) do
+    Class.new do
+      include Procto.call
+
+      def initialize(text)
+        @text = text
+      end
+
+      def call
+        "Hello #{@text}"
+      end
+    end
+  end
+
+  it { should eql('Hello world') }
+
+  it 'exposes a lambda proc' do
+    expect(proc).to be_a_lambda
```

Comment:
Ah I see. Even without that hint from mutant, I now actually fully agree that `to_proc`'s contract should be honored and that there should be a test for that. I didn't think about that at the time I wrote my previous comment and thought that testing the resulting proc's behavior would be enough. I was wrong. Thx for bearing with me!